schema/pet-insurance.json

```diff
@@ -1,680 +1,1338 @@
+                        "Egyptian Mau",
+                        "European Shorthair",
+                        "Exotic Shorthair",
+                        "German Rex",
+                        "Havana Brown",
+                        "Himalayan",
+                        "Japanese Bobtail",
+                        "Khao Manee",
+                        "Korat",
+                        "Korean Bobtail",
+                        "Kurilian Bobtail",
+                        "LaPerm",
+                        "Lykoi",
+                        "Maine Coon",
+                        "Manx",
+                        "Minskin",
+                        "Munchkin",
+                        "Nebelung",
+                        "Napoleon",
+                        "Norwegian Forest Cat",
+                        "Ocicat",
+                        "Ojos Azules",
+                        "Oregon Rex",
+                        "Oriental Bicolor",
+                        "Oriental Shorthair",
+                        "Oriental Longhair",
+                        "Pixie-bob",
+                        "Raas",
+                        "Ragamuffin",
+                        "Russian Blue",
+                        "Savannah",
+                        "Selkirk Rex",
+                        "Serengeti",
+                        "Siamese",
+                        "Thai",
+                        "Siberian",
+                        "Singapura",
+                        "Snowshoe",
+                        "Sokoke",
+                        "Somali",
+                        "Sphynx",
+                        "Suphalak",
+                        "Thai Lilac",
+                        "Tonkinese",
+                        "Toyger",
+                        "Turkish Angora",
+                        "Turkish Van",
+                        "York Chocolate"
+                    ]
+                },
+                "dateOfBirth": {
+                    "type": "string",
+                    "description": "Date of birth in YYYY-MM-DD format.",
+                    "format": "date",
+                    "example": "1976-01-27"
+                },
+                "petPrice": {
+                    "type": "string",
+                    "default": "",
+                    "description": "The price given for the pet."
+                },
+                "relatedQuestions": {
+                    "properties": {
+                        "isSpayedOrNeutered": {
+                            "type": "boolean"
+                        },
+                        "hasChipOrTag": {
+                            "type": "boolean"
+                        },
+                        "isKeptAtYourAddress": {
+                            "type": "boolean"
+                        },
+                        "hasLegalAction": {
+                            "type": "boolean"
+                        },
+                        "hasBehaviourComplains": {
+     
```

Comment:
This should probably be renamed to `CardPayment`, since there is another payment definition in here.